JSDemos/Demos/DateBox/Overview/description.md

```diff
@@ -1,5 +1,25 @@
-The **DateBox** is&nbsp;a&nbsp;UI component that displays the date and time in&nbsp;one of&nbsp;DevExtreme [predefined formats][0] or&nbsp;a&nbsp;format defined in&nbsp;an&nbsp;external localization library, or&nbsp;a&nbsp;type in&nbsp;the required date/time value. In&nbsp;this demo, the &laquo;Custom format&raquo; **DateBox**&rsquo;es value is&nbsp;formatted [using the Globalize library][1]. Alternatively, you can [use Intl][2].
+The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) is an editor that allows users to select date and time and displays them. This demo illustrates the following **DateBox** properties:
 
-[0]: https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/#type
-[1]: /Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Globalize
-[2]: /Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies/Using_Intl
\ No newline at end of file
+- [value](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#value)     
+A value the **DateBox** displays.
+
+- [type](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#type)       
+Specifies whether the **DateBox** allows users to select only date, only time, or both date and time.
+
+- [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat)        
+A date/time display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter case.
+
+- [pickerType](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#pickerType)        
+The type of date/time picker.
```

Comment:
There's already a "type" property described above. Now here's a picker type. In the other item you explained what the "type" was. Here you don't mention what the word "type" means. So maybe we should add some details:

```suggestion
Specifies the type of UI used to select date or time. This demo shows how to change calendar to roller date picker. 
```